source/qml/Samples/QmlVisualizer/AdaptiveCardQml/ActionSetHorizontalRender.qml

```diff
@@ -0,0 +1,63 @@
+import "AdaptiveCardUtils.js" as AdaptiveCardUtils
+import QtQuick 2.3
+
+Flow {
+    id: flowActionId
+
+    property int _spacing
+    property string _layoutDirection
+    property var actionButtonModel
+    property var adaptiveCard
+    property var _toggleVisibilityTarget: null
+    property var activeShowCard: null
+    property var prevLoaderId: null
+    property bool isCentreAlign: false
+    property var rectangleElements: []
+    property var actionElements: []
+
```

Comment:
Why some properties starts with `_` while others are not?

Reply:
its because some of the properties we have to pass to the inner elements whose property names also start with _